### PR TITLE
feat: integrate inspector

### PR DESCRIPTION
**Pull request by cazala**
This PR integrates the `@dcl/inspector`

### CODE REVIEW

File: packages/preload/src/modules/constants.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_THUMNNAIL =
```
Comment (nicoecheza) [inline packages/preload/src/modules/constants.ts:1]: `DEFAULT_THUMBNAIL` :P

File: packages/preload/src/modules/workspace.ts [hunk 2]
```diff
@@ -73,13 +71,14 @@ export async function getProject(_path: string): Promise<Project> {
...
       path: _path,
-      title: scene.display?.title,
+      title: scene.display?.title || 'Untitled scene',
       description: scene.display?.description,
       thumbnail: await getProjectThumbnail(_path, scene),
       layout: getRowsAndCols(parcels),
       createdAt: Number(stat.birthtime),
       updatedAt: Number(stat.mtime),
       size: stat.size,
+      scene,
```
Comment (nicoecheza) [inline packages/preload/src/modules/workspace.ts:81]: does it make sense to return title, description, etc if we are also returning the scene?

File: packages/renderer/src/components/Editor/styles.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  display: flex;
+  flex: 0 0 auto;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 0 16px;
+  height: 48px;
+}
+
+.Editor .header .back {
+  background-image: url("./back.svg");
```
Comment (nicoecheza) [inline packages/renderer/src/components/Editor/styles.css:19]: do we need this icon? I think we can use
`import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';`

File: packages/renderer/src/components/SceneList/component.tsx [hunk 4]
```diff
@@ -129,7 +133,7 @@ export function SceneList({ projects, sortBy, onSort }: Props) {
       </Column>
       <Box
         display="grid"
-        gridTemplateColumns="repeat(4, 1fr)"
+        // gridTemplateColumns="repeat(4, 1fr)"
```
Comment (nicoecheza) [inline packages/renderer/src/components/SceneList/component.tsx:136]: is this ok? I see the UI broken if we remove it

**Review by nicoecheza — CHANGES_REQUESTED** [review]

File: packages/renderer/src/components/SceneList/component.tsx (hunk 4 above)
Comment (cazala) [reply packages/renderer/src/components/SceneList/component.tsx:136]: uh you are right, but this also breaks the empty state. It should be removed only on the empty state but not if you have scenes.

File: packages/preload/src/modules/workspace.ts (hunk 2 above)
Comment (cazala) [reply packages/preload/src/modules/workspace.ts:81]: I's not necessary for now, I'll remove it.

**Review by nicoecheza — APPROVED** [review]

**Pull request merged**